tests/queries/0_stateless/01059_storage_file_compression.sh

```diff
@@ -12,6 +12,7 @@ do
     ${CLICKHOUSE_CLIENT} --query "CREATE TABLE file (x UInt64) ENGINE = File(TSV, '${CLICKHOUSE_DATABASE}/${m}.tsv.${m}')"
     ${CLICKHOUSE_CLIENT} --query "TRUNCATE TABLE file"
     ${CLICKHOUSE_CLIENT} --query "INSERT INTO file SELECT * FROM numbers(1000000)"
+    sleep 1
```

Comment:
@Avogar why this sleep is required?

Reply:
Actually seems that this PR likely breaks performance tests - https://clickhouse-test-reports.s3.yandex.net/0/fc9ef14a736d62f121d27d3a23490ef7198190be/performance_comparison/report.html#fail1

Reply:
>Actually seems that this PR likely breaks performance tests - https://clickhouse-test-reports.s3.yandex.net/0/fc9ef14a736d62f121d27d3a23490ef7198190be/performance_comparison/report.html#fail1

#31565

Reply:
> @Avogar why this sleep is required?

It was just a temporary solution to fix failing test, because I needed these changes in the other PRs. The real fix is in https://github.com/ClickHouse/ClickHouse/pull/31265

Reply:
>>Actually seems that this PR likely breaks performance tests - https://clickhouse-test-?>>reports.s3.yandex.net/0/fc9ef14a736d62f121d27d3a23490ef7198190be/performance_comparison/report.html#fail1
>
>#31565

Thanks